src/app/zap-templates/zcl/data-model/chip/rvc-clean-mode-cluster.xml

```diff
@@ -40,7 +40,6 @@ limitations under the License.
     <!-- Base data types -->
     <attribute side="server" code="0x0000" define="SUPPORTED_MODES"  type="ARRAY" entryType="ModeOptionStruct" writable="false" optional="false" isNullable="false" length="255">SupportedModes</attribute>
     <attribute side="server" code="0x0001" define="CURRENT_MODE"     type="int8u"                              writable="false" optional="false" isNullable="false" reportable="true">CurrentMode</attribute>
-    <attribute side="server" code="0x0002" define="START_UP_MODE"    type="int8u"                              writable="true"  optional="true"  isNullable="true">StartUpMode</attribute>
```

Comment:
I am not an expert in removing attributes at all, so I am asking these questions because I seriously don't know the answer:

* Is there going to be an amendment to the 1.2 spec or is this going in for 1.3 essentially? Will there ever be a device that has this attribute?
* Is there a way for us to mark an attribute at deprecated so that we don't actually use code 2 for anything else in the future.

Reply:
> Is there going to be an amendment to the 1.2 spec or is this going in for 1.3 essentially? 

I am assuming 1.3.

> Is there a way for us to mark an attribute at deprecated so that we don't actually use code 2 for anything else in the future.

This is a derived cluster.  In the spec, it would list the attribute, with "X" for conformance (meaning "not to be implemented in this derived cluster).  See the Bridged Device Basic Information cluster for an example.